dockstore-webservice/src/main/resources/migrations.1.11.0.xml

```diff
@@ -215,4 +215,12 @@
             CREATE UNIQUE INDEX one_token_link_per_identify2 ON token USING btree (username, tokensource) WHERE onlineprofileid IS NULL;
         </sql>
     </changeSet>
+    <changeSet author="natalieperez (generated)" id="fix_onlineprofileid_datatype">
+        <sql dbms="postgresql">
+            UPDATE token SET onlineprofileid = null WHERE true;
+            UPDATE user_profile SET onlineprofileid = null WHERE true;
+        </sql>
+        <modifyDataType columnName="onlineprofileid" newDataType="varchar(255)" tableName="token"/>
+        <modifyDataType columnName="onlineprofileid" newDataType="varchar(255)" tableName="user_profile"/>
```

Comment:
Should there be a foreign key for onlineprofileid between token and user_profile tables? I should have made this comment on the previous PR, but just noticed this time.

Or, maybe not... Will there potentially be onlineprofileids for non-login accounts, e.g., Quay? I might be reading too much into this because of the word "profile".

Reply:
If we wanted, I guess we could get the ids for other linked account options but we don't make rows in the user_profile table for those. We only insert into that table for Google and GitHub. We also let people delete their token for Google and GitHub and end up using what's in the user_profile table to let them continue logging in.